src/vs/workbench/contrib/terminal/browser/terminalInstance.ts

```diff
@@ -1986,33 +2027,33 @@ registerThemingParticipant((theme: IColorTheme, collector: ICssStyleCollector) =
+				.monaco - workbench.hc - black.pane - body.integrated - terminal.xterm.focus:: before,
+			.monaco - workbench.hc - black.pane - body.integrated - terminal.xterm: focus:: before { border - color: ${border}; } `
 		);
 	}
 
 	// Scrollbar
 	const scrollbarSliderBackgroundColor = theme.getColor(scrollbarSliderBackground);
 	if (scrollbarSliderBackgroundColor) {
 		collector.addRule(`
-			.monaco-workbench .pane-body.integrated-terminal .find-focused .xterm .xterm-viewport,
-			.monaco-workbench .pane-body.integrated-terminal .xterm.focus .xterm-viewport,
-			.monaco-workbench .pane-body.integrated-terminal .xterm:focus .xterm-viewport,
-			.monaco-workbench .pane-body.integrated-terminal .xterm:hover .xterm-viewport { background-color: ${scrollbarSliderBackgroundColor} !important; }
-			.monaco-workbench .pane-body.integrated-terminal .xterm-viewport { scrollbar-color: ${scrollbarSliderBackgroundColor} transparent; }
-		`);
+				.monaco - workbench.pane - body.integrated - terminal.find - focused.xterm.xterm - viewport,
+			.monaco - workbench.pane - body.integrated - terminal.xterm.focus.xterm - viewport,
+			.monaco - workbench.pane - body.integrated - terminal.xterm: focus.xterm - viewport,
+			.monaco - workbench.pane - body.integrated - terminal.xterm: hover.xterm - viewport { background - color: ${scrollbarSliderBackgroundColor} !important; }
+			.monaco - workbench.pane - body.integrated - terminal.xterm - viewport { scrollbar - color: ${scrollbarSliderBackgroundColor} transparent; }
+			`);
 	}
 
 	const scrollbarSliderHoverBackgroundColor = theme.getColor(scrollbarSliderHoverBackground);
 	if (scrollbarSliderHoverBackgroundColor) {
 		collector.addRule(`
-			.monaco-workbench .pane-body.integrated-terminal .xterm .xterm-viewport::-webkit-scrollbar-thumb:hover { background-color: ${scrollbarSliderHoverBackgroundColor}; }
-			.monaco-workbench .pane-body.integrated-terminal .xterm-viewport:hover { scrollbar-color: ${scrollbarSliderHoverBackgroundColor} transparent; }
-		`);
+				.monaco - workbench.pane - body.integrated - terminal.xterm.xterm - viewport:: -webkit - scrollbar - thumb: hover { background - color: ${scrollbarSliderHoverBackgroundColor}; }
+			.monaco - workbench.pane - body.integrated - terminal.xterm - viewport: hover { scrollbar - color: ${scrollbarSliderHoverBackgroundColor} transparent; }
+			`);
 	}
 
 	const scrollbarSliderActiveBackgroundColor = theme.getColor(scrollbarSliderActiveBackground);
 	if (scrollbarSliderActiveBackgroundColor) {
-		collector.addRule(`.monaco-workbench .pane-body.integrated-terminal .xterm .xterm-viewport::-webkit-scrollbar-thumb:active { background-color: ${scrollbarSliderActiveBackgroundColor}; }`);
+		collector.addRule(`.monaco - workbench.pane - body.integrated - terminal.xterm.xterm - viewport:: -webkit - scrollbar - thumb: active { background - color: ${scrollbarSliderActiveBackgroundColor}; } `);
```

Comment:
This thing again 😱, has happened to me a couple of times too

Reply:
I disabled prettier and had fixed that... I don't know how it happened again 😦

Reply:
Ah, this is the cause 65e1707d19d4df5b1500ae260ceae9fad6d0cd96. I'm going to disable as I'm not a fan of format on save

Reply:
I think you might be looking at an old version? I fixed that earlier I think.